haroldtreen/readmeio-sync
# Upload Doc/Category order from syncRegistry.json

# Ordering Upload

Status: **Ready for Review**

Reviewers: @donnielrt @fractaltheory 
JIRA: [RTM-224](https://mobify.atlassian.net/browse/RTM-224)
Linked PRs: https://github.com/mobify/readmeio-sync/pull/4
## Changes
- Adds a function for uploading documentation ordering
- Calls the ordering upload when doing `readmeio-sync upload`
## Test Drive
- grunt test
- Read the README to understand the the tool and commands (https://github.com/mobify/readmeio-sync/blob/enforce-order/README.md)
- Set your environment variables for `README_EMAIL` and `README_PASSWORD`
- Configure a staging and production slug (they can be the same) using `readmeio-sync config`
- Initialize the project using `readmeio-sync init`
- Change the order of documents in `syncRegistry.json`
- Upload the changes using `readmeio-sync upload`
- Confirm the ordering changes have propagated
